### PR TITLE
[Monitor] az monitor autoscale: support namespace and dimensions for --condition

### LINKED ISSUE #14126: Enable dimension for Azure Monitor auto-scale rule
**Resource Provider**

Microsoft.Insights/autoscaleSettings

**Description of Feature or Work Requested**

https://github.com/Azure/azure-rest-api-specs/pull/9945

**Minimum API Version Required**

2015-04-01

**Swagger Link**

https://github.com/Azure/azure-rest-api-specs/pull/9945

**Target Date**

TBD

### PR DESCRIPTION
**Pull request by MyronFanQiu**
**Description**  

Fix #14126 

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Monitor

**Comment by qwordy** [comment]
The description part is kind of simple.

File: src/azure-cli/azure/cli/command_modules/monitor/grammar/autoscale/AutoscaleConditionLexer.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        buf.write(u"\2\2\2\u00d3\u00d4\3\2\2\2\u00d4\u00d5\3\2\2\2\u00d5")
+        buf.write(u"\u00d8\7\f\2\2\u00d6\u00d8\7\17\2\2\u00d7\u00d3\3\2\2")
+        buf.write(u"\2\u00d7\u00d6\3\2\2\2\u00d8\u00d9\3\2\2\2\u00d9\u00d7")
+        buf.write(u"\3\2\2\2\u00d9\u00da\3\2\2\2\u00daR\3\2\2\2\u00db\u00e0")
+        buf.write(u"\5;\36\2\u00dc\u00e0\5=\37\2\u00dd\u00e0\59\35\2\u00de")
+        buf.write(u"\u00e0\7a\2\2\u00df\u00db\3\2\2\2\u00df\u00dc\3\2\2\2")
+        buf.write(u"\u00df\u00dd\3\2\2\2\u00df\u00de\3\2\2\2\u00e0\u00e1")
+        buf.write(u"\3\2\2\2\u00e1\u00df\3\2\2\2\u00e1\u00e2\3\2\2\2\u00e2")
+        buf.write(u"T\3\2\2\2\r\2\u00bc\u00c1\u00c7\u00c9\u00d0\u00d3\u00d7")
+        buf.write(u"\u00d9\u00df\u00e1\2")
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/monitor/grammar/autoscale/AutoscaleConditionLexer.py:110]: Magic

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/monitor/grammar/autoscale/AutoscaleConditionLexer.py:110]: What is it?

Comment (MyronFanQiu) [reply src/azure-cli/azure/cli/command_modules/monitor/grammar/autoscale/AutoscaleConditionLexer.py:110]: This is auto-generated by [Antrl4](https://www.antlr.org/) tool.

**Pull request merged**